lib/bokken/birthdaynotifier.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Bokken.BirthdayNotifier do
```

Comment:
The filename should be `birthday_notifier.ex`.
> Generally speaking, filenames follow the snake_case convention of the module they define. For example, MyApp should be defined inside the my_app.ex file.

https://hexdocs.pm/elixir/1.12/naming-conventions.html